client/post-editor/editor-featured-image/index.jsx

```diff
@@ -159,6 +166,8 @@ class EditorFeaturedImage extends Component {
 					</Button>
 					{ featuredImageId ? <RemoveButton onRemove={ EditorFeaturedImage.removeImage } /> : '' }
 				</div>
+
+				{ this.props.hasDropZone ? <FeaturedImageDropZone /> : '' }
```

Comment:
You should be able to use the conventional:
`this.props.hasDropZone && <FeaturedImageDropZone />;`
...instead of the ternary here as null values should be ignored.

Reply:
Even when keeping this construct, `null` would be preferable over `""` an empty string

Reply:
Ah, good tips, I'll incorporate that before rebasing. Thanks!